include/wget/wget.h

```diff
@@ -265,6 +265,8 @@ int
 	wget_global_get_int(int key) LIBWGET_EXPORT;
 void
 	(*wget_global_get_func(int key))(const char *, size_t) LIBWGET_EXPORT;
+void
+	wget_handle_sigwinch(void) LIBWGET_EXPORT;
```

Comment:
Then I think I understood you just fine. See the commit 354c862. It renames the function as you suggest to make it clearer.